src/main/java/core/util/HelperWrapper.java

```diff
@@ -40,76 +40,76 @@ public static HelperWrapper instance() {
         return m_clInstance;
     }
 
-    public Date getHattrickDate(String string) throws ParseException {
-        final SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd hh:mm:ss", Locale.UK);
-        final Date d = sdf.parse(string);
-        final String hh = string.substring(11, 13);
-
-        if (hh.equalsIgnoreCase("12")) {
-            final Calendar c = Calendar.getInstance(Locale.UK);
-            c.setFirstDayOfWeek(Calendar.SUNDAY);
-            c.setTime(d);
-            c.add(Calendar.HOUR_OF_DAY, 12);
-            return c.getTime();
-        }
-
-        return d;
-    }
-
-    /**
-     * Calculate the last valid training date for a certain date (skillupDate)
-     *
-     * @param skillupDate the skillupdate or HRF date
-     * @param refTrainingDate a reference containing a valid training time and day of week
-     *
-     * @return the last valid training date for the given 'skillupDate'
-     */
-    public Calendar getLastTrainingDate(Date skillupDate, Date refTrainingDate) {
-        // Calendar for TrainingDate
-        final Calendar trDate = Calendar.getInstance(Locale.US);
-
-        // IF refTrainingDate is null first run of HO
-        if (refTrainingDate == null) {
-            return trDate;
-        }
-
-        trDate.setTime(refTrainingDate);
-        trDate.setFirstDayOfWeek(Calendar.SUNDAY);
-        trDate.setLenient(true);
-        trDate.setMinimalDaysInFirstWeek(1);
-
-        // Calendar for Skillup Date
-        final Calendar suDate = Calendar.getInstance(Locale.US);
-        suDate.setFirstDayOfWeek(Calendar.SUNDAY);
-        suDate.setMinimalDaysInFirstWeek(1);
-        suDate.setLenient(true);
-        suDate.setTime(skillupDate);
-
-        // Move TrainingDate back to skillup week
-        trDate.set(Calendar.YEAR, suDate.get(Calendar.YEAR));
-        trDate.set(Calendar.WEEK_OF_YEAR, suDate.get(Calendar.WEEK_OF_YEAR));
-
-        // Check that is fine
-        long diff = suDate.getTimeInMillis() - trDate.getTimeInMillis();
-
-        // training date must be within one week (handle end of year)
-        while (diff > WEEK) {
-        	trDate.add(Calendar.WEEK_OF_YEAR, +1);
-        	diff = suDate.getTimeInMillis() - trDate.getTimeInMillis();
-        }
-
-        // training date must not be after skillup date
-        while (diff != 0 && trDate.after(suDate)) {
-        	trDate.add(Calendar.WEEK_OF_YEAR, -1);
-        }
-
-        final Calendar c = Calendar.getInstance(Locale.UK);
-        c.setFirstDayOfWeek(Calendar.SUNDAY);
-        c.setMinimalDaysInFirstWeek(1);
-        c.setTime(trDate.getTime());
-        return c;
-    }
-
+//    public Date getHattrickDate(String string) throws ParseException {
```

Comment:
Nice cleanup, just remove if that's not needed any more ;-)

Reply:
removed